client/main.lua

```diff
@@ -356,7 +356,7 @@ CreateThread(function()
                 sleep = 2000
                 local plate = QBCore.Functions.GetPlate(entering)
                 QBCore.Functions.TriggerCallback('vehiclekeys:server:CheckOwnership', function(result)
-                    if not result then -- if not player owned
+                    if result ~= true then -- if not player owned
```

Comment:
'result ~= true' and 'not result' is the same , would suggest changing back to changing result.

Reply:
Yeah, Ik. Changing it to not result